config/routes.rb

```diff
@@ -68,6 +69,7 @@
             when :delete
               delete "/:c_id/#{subcollection_name}/:s_id", :action => :destroy
             when :post
+              post "/:c_id/#{subcollection_name}", :action => :create, :constraints => Api::CreateConstraint.new
```

Comment:
I wonder if we can use similar constraints to deal with the exploding number of routes on the UI side 🤔  cc @jrafanie

Reply:
Hopefully... I don't really know routes that well.